doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -597,4 +597,4 @@ cJSON
 Documentation
 =============
 
-|no_changes_yet_note|
+* Added a new page describing the :ref:`sQSPI MSPI shim <mspi_sqspi>` driver.
```

Comment:
I'd say this entry belongs under Drivers.

Reply:
Agreed. I'll move it there.

Reply:
Done.